scripts/verify-sha256sum-signature

```diff
@@ -5,27 +5,17 @@
 # We expect a valid signature to correspond to a SecureDrop Maintainer,
 # so we create a temporary keyring in order to force gpg to use *only*
 # authorized keys during the verification check.
-set -e
-set -u
-set -o pipefail
-set -o nounset
+set -euo pipefail
```

Comment:
The `x` is for verbose output that prints all the commands as they're executed. I felt it would be a bit too verbose for this script (which I think is probably already too verbose since it prints all the gpg output...) but if you disagree happy to add it in.

Reply:
Makes sense, if it's intentional I trust your judgment 👍